packages/apps/types/src/schema/thread.ts

```diff
@@ -28,14 +29,14 @@ export class MessageType extends TypedObject({ typename: 'dxos.org/type/Message'
   text: S.String,
   /** Non-text content sent with a message (e.g. files, polls, etc.) */
   parts: S.optional(S.mutable(S.Array(ref(Expando)))),
-  /** Context of the application when message was created. */
-  // TODO(burdon): Evolve "attention object" to be current UX state? E.g., of Deck?
-  context: S.optional(ref(Expando)),
   /** Custom properties for specific message types (e.g. email subject or cc fields). */
   properties: S.optional(S.mutable(S.Record(S.String, S.Any))),
   // TODO(wittjosiah): Add read status:
```

Comment:
this todo should probably stay at the end otherwise it reads as though it applies to the field below it